src/tracer.js

```diff
@@ -207,19 +206,17 @@ export default class Tracer {
      * Returns a SpanContext instance extracted from `carrier` in the given
      * `format`.
      *
-     * OpenTracing defines a common set of `format` values (see FORMAT_TEXT_MAP
-     * and FORMAT_BINARY), and each has an expected carrier type.
+     * OpenTracing defines a common set of `format` values (see
+     * FORMAT_TEXT_MAP, FORMAT_HTTP_HEADERS, and FORMAT_BINARY), and each has
+     * an expected carrier type.
      *
      * Consider this pseudocode example:
      *
      *     // Use the inbound HTTP request's headers as a text map carrier.
-     *     var textCarrier = inboundHTTPReq.headers;
-     *     var wireCtx = Tracer.extract(Tracer.FORMAT_TEXT_MAP, textCarrier);
+     *     var headersCarrier = inboundHTTPReq.headers;
+     *     var wireCtx = Tracer.extract(Tracer.FORMAT_HTTP_HEADERS, headersCarrier);
      *     var serverSpan = Tracer.startSpan('...', Tracer.childOf(wireCtx));
```

Comment:
Also not new, but noticing it: should this be ` Tracer.startSpan('...',  { childOf : wireCtx });`